### ISSUE TITLE
SWR

### ISSUE THREAD

**Issue by pentiger** [sprakashb/FT2000-CAT-Library#1]
Hi, can this be used to read SWR from the transceiver to use in the magnetic loop controller?

**Comment by sprakashb** [comment]
If the rig has command to read the swr then we can. Could you please check
the rigs manual.

Currently it is not implemented but we can easily add the command, if
available in rig.

73s
Vu2spf Bhatnagar

**Comment by pentiger** [comment]
https://www.yaesu.com/airband/downloadFile.cfm?FileID=16477&FileCatID=158&FileName=FTDX3000%5FCAT%5FOM%5FENG%5F2006%2DD.PDF&FileContentType=application%2Fpdf

According to this the command is RM.

![image](https://user-images.githubusercontent.com/13071060/110265068-b7270680-7f88-11eb-91fc-5834901ff220.png)

**Comment by sprakashb** [comment]
Thanks, I'll check it out.

73's

VU2SPF / Bhatnagar

**Comment by pentiger** [comment]
No, thank you!!! I believe many people would benefit from this feature. 

73!
KC3QYG

**Comment by sprakashb** [comment]
Do you have a compatible rig to test the program. I do not have any
available.

I can send the modified program for you to upload on Arduino connected to a
rig.

73s

Vu2spf Bhatnagar

**Comment by pentiger** [comment]
Hi, that would be great. I do have FT-DX3000, I'll just have to make the
CAT cable, and I can test it.


*Piotr Bagniewski / KC3QYG <https://www.qrz.com/lookup/KC3QYG> */ *HH 14735
<https://apps.wizworks.net:9091/results.php> */ *HH Cell 14717
<https://apps.wizworks.net:9091/results.php>*
Install the fastest browser: https://brave.com/pbm645

**Comment by sprakashb** [comment]
That's is great news. Give me a few days.

73s
Vu2spf
